Backend/app/rider-platform/rider-app/Main/src/Domain/Action/UI/Call.hs

```diff
@@ -232,6 +235,9 @@ getDriverMobileNumber callSid callFrom_ callTo_ _dtmfNumber callStatus to_ = do
             createdAt = now,
             updatedAt = now
           }
+    getNumberBasedOnType numberType ride = case numberType of
```

Comment:
@hkmangla bro, 

 
```haskell
AlternateNumber -> fromMaybe ride.driverMobileNumber ride.driverAlternateNumber
PrimaryNumber -> ride.driverMobileNumber
``` 

here on AlternateNumber case should i return PrimaryNumber on not getting Alternate Number?